### ISSUE TITLE
Will editing the values file for your library cause a issue within your library?

### ISSUE THREAD

**Issue by karanthakakr04** [Jay-Goo/RangeSeekBar#5]
I have this person working with me who is saying that if we edit the values file in the output folder in the main library (here on GitHub) get's reflected with changes and it could cause problems with the library.

For instance I wanted 3 seekbars and I needed to set checkpoints for each of them so what I did is I added this piece of code,

```
<string-array name="markArray2">
        <item>6</item>
        <item>12</item>
        <item>18</item>
        <item>24</item>
    </string-array>
    <string-array name="markArray3">
        <item>3</item>
        <item>6</item>
        <item>9</item>
        <item>12</item>
    </string-array>
```
Similar code which comes when library is default run first time. Did I do it wrong?

**Comment by JinJieGu** [comment]
Do you want 3 seekbars and set different checkpoints for each?
If you do,you can see more detail form **markTextArray** and **cellMode**
if you want konw more,you can see the demo [layout.xml](https://github.com/Jay-Goo/RangeSeekBar/blob/master/app/src/main/res/layout/activity_main.xml)
Do I understand you ?

**Comment by karanthakakr04** [comment]
No buddy. I already know how to use your library. I just posted this issue because one of my colleagues was telling me that if I edit `values.xml` file which is present in the path `/private/var/root/.android/build-cache/9bdad5672104325f610db6b35dd1ab5a94c2cf6d/output/res/values/values.xml`. It will cause problems with the main library on GitHub, that is here. As I included the dependency of your library in my gradle and started using it. 
I know the above writing much be confusing for you. So this is a succinct gist of whatever I posted.
